### PR TITLE
fix: Using 3 letter ISO639 language codes

### PR DESCRIPTION
**Pull request by jlacivita**

### CODE REVIEW

File: src/openrpc/accessibility.json [hunk 1]
```diff
@@ -51,7 +51,11 @@
                 "textAlignVertical": "middle",
                 "windowColor": "white",
                 "windowOpacity": 50
-              }
+              },
+              "preferredLanguages": [
+                "eng",
+                "esp"
```
Comment (DeltaHeadMan) [inline src/openrpc/accessibility.json:57]: "esp" should be "spa"

File: src/openrpc/accessibility.json [hunk 2]
```diff
@@ -100,7 +104,11 @@
...
```
Comment (DeltaHeadMan) [inline src/openrpc/accessibility.json:110]: Change "esp" to "spa"

File: src/openrpc/closed_captions.json [hunk 3]
```diff
@@ -621,6 +621,50 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "preferredLanguages",
+            "summary": "A prioritized list of ISO 639 1/2 codes for the preferred closed captions languages on this device.",
```
Comment (DeltaHeadMan) [inline src/openrpc/closed_captions.json:627]: State that it's ISO 639-2/B

File: src/openrpc/closed_captions.json [hunk 4]
```diff
@@ -621,6 +621,50 @@
...
+                ]
+              }
+            ],
+            "result": {
+              "name": "languages",
+              "summary": "the preferred closed captions languages",
+              "schema": {
+                "type": "array",
+                "items": {
+                  "$ref": "https://meta.comcast.com/firebolt/localization#/definitions/ISO639Language"
```
Comment (DeltaHeadMan) [inline src/openrpc/closed_captions.json:646]: Suggest to call this: ISO639-2Language

File: src/openrpc/closed_captions.json [hunk 5]
```diff
@@ -621,6 +621,50 @@
...
+                }
+              }
+            },
+            "examples": [
+              {
+                "name": "Default Example",
+                "params": [],
+                "result": {
+                  "name": "Default Result",
+                  "value": ["esp", "eng"]
```
Comment (DeltaHeadMan) [inline src/openrpc/closed_captions.json:656]: "esp" needs to be "spa"

File: src/openrpc/closed_captions.json [hunk 6]
```diff
@@ -621,6 +621,50 @@
...
+                }
+              }, 
+              {
+                "name": "Default Example #2",
+                "params": [],
+                "result": {
+                  "name": "Default Result",
+                  "value": ["eng", "esp"]
```
Comment (DeltaHeadMan) [inline src/openrpc/closed_captions.json:664]: Ditto

File: src/openrpc/localization.json [hunk 7]
```diff
@@ -99,7 +99,7 @@
           ]
         }
       ],
-      "summary": "Get the ISO 3166 code for the counrty device is located in",
+      "summary": "Get the ISO 316601 alpha-2 code for the counrty device is located in",
```
Comment (DeltaHeadMan) [inline src/openrpc/localization.json:102]: "ISO 3166-1 alpha-2"

Comment (DeltaHeadMan) [inline src/openrpc/localization.json:102]: typo: counrty should be country

File: src/schemas/localization.json [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    "definitions": {
+        "CountryCode": {
+            "type": "string",
+            "pattern": "^[A-Z]{2}$"
+        },
+        "Language": {
+            "type": "string",
+            "pattern": "^[A-Za-z]{2}$"
+        },
+        "ISO639Language": {
```
Comment (DeltaHeadMan) [inline src/schemas/localization.json:30]: ISO639-2Language, I suggest.

File: src/openrpc/localization.json [hunk 9]
```diff
@@ -132,6 +132,11 @@
       "summary": "Get the ISO 639 1/2 code for the preferred language",
       "params": [],
       "tags": [
+        {
+          "name": "depecated",
```
Comment (DeltaHeadMan) [inline src/openrpc/localization.json:136]: Typo:  depecated -> deprecated

File: src/openrpc/localization.json [hunk 10]
```diff
@@ -132,6 +132,11 @@
       "summary": "Get the ISO 639 1/2 code for the preferred language",
       "params": [],
       "tags": [
+        {
```
Comment (DeltaHeadMan) [inline src/openrpc/localization.json:135]: also deprecate setLangauge and onLanguageChanged

Comment (jlacivita) [reply src/openrpc/localization.json:135]: These are autogenerated, and will inherit the deprecation upon generation.

File: src/openrpc/closed_captions.json (hunk 4 above)
Comment (jlacivita) [reply src/openrpc/closed_captions.json:646]: We've got an undocumented requirement that all schema names are safe variable names for JS & C.

Probably worth fixing in our code generation, but not on a Friday. Happy to look into it Monday if you still want the hyphen, but made it an underscore for now.

`ISO639_2Language`

File: src/schemas/localization.json (hunk 8 above)
Comment (jlacivita) [reply src/schemas/localization.json:30]: Same response as above

**Review by kevinshahfws — APPROVED** [review]

**Pull request merged**